### DISCUSSION TITLE
Verified Servers

### DISCUSSION THREAD

**Discussion by austin4195** [ServerListPlus/issues#17] [Ideas]
Not sure if this is a good idea but are there plans to implement some sort of Verified status where if a server owner is registered in the reforged server owners discord, they receive this mark?

Just an idea I am throwing out!

**Comment by Brend-Smits** [comment]
We thought about a verification system but weren't sure what it would benefit users. 
To the end-user, what does it mean that a server or user is verified? Should it give the user more trust?

**Reply by austin4195** [reply]
So verified would just mean that they have been “checked” by reforged staff.

in terms of other benefits, see issue #18

**Reply by Brend-Smits** [reply]
That makes sense. Since this is not an official Pixelmon server list, or an official Reforged server list, I would have to check with Ras and see if this is something we could do, or even want to do. 
Do you have any other ideas on server verification?

**Reply by austin4195** [reply]
Ah, It was my understanding that this was an official server list. Anywho, besides manual verification (checking does a website actually exist and is the server really who they say they are) that was all I had on ideas for verification. Ras/Xpand usually checks those things prior to allowing access so that's why I suggested automatically giving the verified tag to those in Reforged Servers.
